proto/arkeo/claim/tx.proto

```diff
@@ -13,31 +14,31 @@ service Msg {
   // this line is used by starport scaffolding # proto/tx/rpc
 }
 message MsgClaimEth {
-  string creator = 1;
+  bytes creator = 1 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
   string eth_address = 2; // the adress the claim is for
   string signature = 3; // EIP712 signature that has to be signed by ethAddress
 }
 
 message MsgClaimEthResponse {}
 
-message MsgClaimArkeo { string creator = 1; }
+message MsgClaimArkeo { bytes creator = 1 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ]; }
 
 message MsgClaimArkeoResponse {}
 
 message MsgTransferClaim {
-  string creator = 1;
-  string toAddress = 2;
+  bytes creator = 1 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
+  bytes toAddress = 2 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
 }
 
 message MsgTransferClaimResponse {}
 
 message MsgAddClaim {
-  string creator = 1;
+  bytes creator = 1 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
   Chain chain = 2;
-  string address = 3;
+  bytes address = 3 [ (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress" ];
```

Comment:
not a huge deal, but for this one, the address here can also be an ETH addy.  this is just for testing purposes as this TX type only works on testnet, but may be nice functionality when people are developing the claim UI so we don't have to worry about creating claims on testnet and people can self serve.